app/src/engine/uci_engine.cpp

```diff
@@ -254,7 +254,7 @@ void UciEngine::sendSetoption(const std::string &name, const std::string &value)
     auto option = uci_options_.getOption(name);
 
     if (!option.has_value()) {
-        Logger::info<true>("Warning; {} doesn't have option {}", config_.name, name);
+        Logger::info<true>("Warning; {} does not have option {}", config_.name, name);
```

Comment:
fishtest reads this line and checks for "doesn't have option" so that would also need a change